### PR TITLE
new icon: consul (original, original-wordmark, plain-wordmark)

### LINKED ISSUE #1603: [ICON REQUEST]: HashiCorp Consul
### I have searched through the issues and didn't find my problem.

- [X] Confirm

### About the icon

Consul uses service identities and traditional networking practices to help organizations securely connect applications running in any environment.

### Links and sources

Project Homepage: https://www.consul.io/
HashiCorp Brand Guidelines: https://www.hashicorp.com/brand
Logo: https://drive.google.com/uc?export=download&id=17tlD38R-KQmAN1NnQ0RL6ROeXOEwiWRx

### Additional information

Just splitting from issue https://github.com/devicons/devicon/issues/1006

### PR DESCRIPTION
**Pull request by luizcarloscf**
## Double check these details before you open a PR

- [x] PR does not match another non-stale PR currently opened
- [x] PR name matches the format *new icon: <i>Icon name</i> (<i>versions separated by comma</i>)*. More details [here](https://github.com/devicons/devicon/blob/develop/CONTRIBUTING.md#overview)
- [x] PR's base is the `develop` branch.
- [x] Your icons are inside a folder as seen [here](https://github.com/devicons/devicon/blob/develop/CONTRIBUTING.md#organizational-guidelines)
- [x] SVG matches the standards laid out [here](https://github.com/devicons/devicon/blob/develop/CONTRIBUTING.md#svgStandards)
- [x] A new object is added in the `devicon.json` file as seen [here](https://github.com/devicons/devicon/blob/develop/CONTRIBUTING.md#-updating-the-deviconjson-)

**This PR closes #1603**

## Link to prove your SVG is correct and up-to-date.

[Project Homepage](https://www.consul.io/)
[HashiCorp Brand Guidelines](https://www.hashicorp.com/brand)
[Logo](https://drive.google.com/uc?export=download&id=17tlD38R-KQmAN1NnQ0RL6ROeXOEwiWRx)

### CODE REVIEW

File: devicon.json [hunk 1]
```diff
@@ -1689,6 +1689,32 @@
             }
         ]
     },
+    {
+        "name": "consul",
+        "altnames": [],
+        "tags": [
+            "networking",
+            "infrastructure"
+        ],
```
Comment (lunatic-fox) [inline devicon.json:1698]: Two more tags.
```suggestion
        "tags": [
            "networking",
            "infrastructure",
            "security",
            "tool"
        ],
```

**Review by lunatic-fox — CHANGES_REQUESTED** [review]
Hello there!
Since `original` version is already a monochromatic icon I took the liberty to remove `plain` version and aliased `original` as `plain` in `devicon.json`. Hope that's ok! 😁

I also suggesting two more tags `security` and `tool`.

You've made a great pull request! 🚀

**Comment by luizcarloscf** [comment]
Hey @lunatic-fox, I think I added all the requested changes. Also, the suggestions you made about original version being already a monochromatic icon and making alias of original as plain in devicon.json apply to another PR that I made #1609.

I've dedicated myself to contributing good quality icons, is there anything that could be better?

**Review by lunatic-fox — DISMISSED** [review]
Hello @luizcarloscf, perfect catch on your other PR.

Well, I think you're already doing a great job. If you want to get more specific info you can take a look at [**SVG Versions**](https://github.com/devicons/devicon/wiki/SVG-Versions) and [**Updating `devicon.json`**](https://github.com/devicons/devicon/wiki/Updating-%60devicon.json%60) wiki topics.

There's also the [**Devicon Discord server**](https://discord.gg/hScy8KWACQ).

A little thing about GitHub is that is possible to commit someone's suggestion this way:
- [Docs GitHub - Incorporating feedback in your pull request](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/incorporating-feedback-in-your-pull-request).

Thank you for this contribution! ✔🚀

File: devicon.json [hunk 2]
```diff
@@ -1689,6 +1689,34 @@
             }
         ]
     },
+    {
+        "name": "consul",
+        "altnames": [],
```
Comment (Snailedlt) [inline devicon.json:1694]:
```suggestion
        "altnames": ["HashiCorp Consul"],
```

**Review by Snailedlt — CHANGES_REQUESTED** [review]

**Comment by Snailedlt** [comment]
> Hey @lunatic-fox, I think I added all the requested changes. Also, the suggestions you made about original version being already a monochromatic icon and making alias of original as plain in devicon.json apply to another PR that I made #1609.
> 
> I've dedicated myself to contributing good quality icons, is there anything that could be better?

The icons are excellent! 💯 

I think the only thing to make them better is to add a new line at the end of each file, since it's needed for some older text editors and IDE's.
See this warning: 
image

**Review by Snailedlt — CHANGES_REQUESTED** [review]
Resizing looks good, however there are now two newlines instead of one at the end of each file. It's a bit confusing since GitHub automatically hides the last newline, but if you edit the file, you'll see that there are 2 empty lines at the end.

So make sure there's only one newline at the end of each file :)

File: devicon.json [hunk 3]
```diff
@@ -1689,6 +1689,34 @@
             }
         ]
     },
+    {
+        "name": "consul",
+        "altnames": ["HashiCorp Consul"],
```
Comment (lunatic-fox) [inline devicon.json:1694]:
```suggestion
        "altnames": [
            "HashiCorp Consul"
        ],
```

**Comment by luizcarloscf** [comment]
That's kind odd, I opened with a text editor and I saw only two lines in the SVG files

**Comment by luizcarloscf** [comment]
I opened the icon files with `vim` and use the command `:set list` to display tabs and newline characters.
I hope that everything is fine now :smiley:

**Review by lunatic-fox — APPROVED** [review]
Approved! ✔
Thank you for this great contribution! 🚀

**Review by Snailedlt — APPROVED** [review]
LGTM! ✔️

**Comment by Snailedlt** [comment]
Thanks for another great PR, and thank you for making our review process easy by having high quality PR's!

**Pull request merged**